### PR TITLE
Propagate Click Event to Res Viewer Comp.

### PR DESCRIPTION
**Pull request by tobiasschweizer**
resolves DSP-784

How to test this in the playground:
- add a standoff link to the text value on the modify page:
Screenshot 2020-11-04 at 17 26 38
- from the "Viewer Module" tab you can now hover over or click standoff links (test) and links (Sierra) and get some console logs

### CODE REVIEW

**Comment by tobiasschweizer** [comment]
after discussion with @mdelez: the editing / creation of a standoff link in a text can lead to the creation of a standoff link managed by the system. We have to find a way to figure out how to get the standoff link when modifying a text value with markup

**Comment by tobiasschweizer** [comment]
after discussion with @mdelez: the required logic for getting standoff link values from the resource will be implemented on a different PR so this PR can be reviewed soon.

File: projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts [hunk 1]
```diff
@@ -72,6 +73,10 @@ export class ResourceViewComponent implements OnInit, OnChanges, OnDestroy {
      */
     @Output() openProject: EventEmitter<ReadProject> = new EventEmitter<ReadProject>();
 
+    @Output() referredResourceClicked: EventEmitter<ReadLinkValue> = new EventEmitter<ReadLinkValue>();
+
+    @Output() referredResourceHovered: EventEmitter<ReadLinkValue> = new EventEmitter<ReadLinkValue>();
```
Comment (kilchenmann) [inline projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts:78]: Maybe we should rename the `openProject` output to a similar name as you did e.g. `referredProjectClicked`. We should keep it consistent and I like the name `referred...Clicked`. I will create a task. We shouldn't do it in this PR

Comment (kilchenmann) [reply projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts:78]: Done in DSP-1023

**Review by kilchenmann — APPROVED** [review]

File: projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts (hunk 1 above)
Comment (tobiasschweizer) [reply projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts:78]: If you think that `referred` is a bit too verbose we could also consistently change it (being aware of breaking the public API of course).

Comment (kilchenmann) [reply projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts:78]: No it's fine

**Pull request merged**